app/services/weets.js

```
@@ -1,12 +1,30 @@
 const api = require('../../config/axios');
+const db = require('../models');
 const errors = require('../errors');
 
-const fetchWeet = () =>
-  api
-    .get('/random')
-    .then(({ data }) => data)
-    .catch(() => Promise.reject(errors.databaseError('Could not connect with external api')));
+const fetchWeet = async () => {
+  try {
+    const { data } = await api.get('/random');
+    if (!data) throw errors.databaseError('Could not connect with external api');
```

Here you should throw an error related to the External API